project_national/fact_sheets_category_3.yml

```diff
@@ -0,0 +1,354 @@
+  skip_sims: true
+
+eagle:
+  n_jobs: 1
+  minutes_per_sim: 4
+  account: abctypol
+  postprocessing:
+    time: 20
+    n_workers: 1
+  sampling:
+    time: 5
+
+postprocessing:
+  keep_individual_timeseries: false
+  aws:
+    region_name: us-west-2
+    s3:
+      bucket: largeee
+      prefix: test_runs
+    athena:
+      glue_service_role: service-role/AWSGlueServiceRole-default
+      database_name: largeee_test_runs
+      max_crawling_time: 1200
+
+workflow_generator:
+  type: residential_hpxml
+  args:
+    build_existing_model:
+      simulation_control_timestep: 60
+      simulation_control_run_period_begin_month: 1
+      simulation_control_run_period_begin_day_of_month: 1
+      simulation_control_run_period_end_month: 12
+      simulation_control_run_period_end_day_of_month: 31
+      simulation_control_run_period_calendar_year: 2007
+
+    emissions:
+      - scenario_name: LRMER_MidCase_15
+        type: CO2e
+        elec_folder: data/cambium/LRMER_MidCase_15
+
+    utility_bills:
+      - scenario_name: Bills
+
+    simulation_output_report:
+      timeseries_frequency: timestep #monthly
+      include_timeseries_total_consumptions: true
+      include_timeseries_fuel_consumptions: true
+      include_timeseries_end_use_consumptions: true
+      include_timeseries_emissions: true
+      include_timeseries_total_loads: false
+
+    reporting_measures:
+      - measure_dir_name: QOIReport
+
+    server_directory_cleanup:
+      retain_in_idf: false
+      retain_schedules_csv: false
+
+### -- Upgrades ---
+upgrades:
+  - upgrade_name: All ENERGYSTAR Appliances
+    options:
+      - &energystar_refrigerator
+        option: Refrigerator|EF 19.9
+        apply_logic:
+          - or:
+            - Refrigerator|EF 6.7
+            - Refrigerator|EF 10.2
+            - Refrigerator|EF 10.5
+            - Refrigerator|EF 15.9
+            - Refrigerator|EF 17.6
+        costs:
+          - value: 2003.0 # price1: $1699-1799 for ENERGY STAR Certified 20.9 Cu.Ft. Refrigerator in BestBuy
+                          # price2: $8144-8599 for ENERGY STAR Certified 20.9 Cu.Ft. Refrigerator in Home Depot. This price is too high. Therefore, will not refer these prices. 
+                          # 2003.0=1749*1.07+94*1.4, price*1.07 sales tax + 94*1.4 labor. 1749=(1699+1799)/2
+                          # Another option: $938. $670 for EF 19.9 20.9 Cu.Ft. Refrigerator in REMDB, then apply 1.4 inflation factor (Jan 2010 to Apr 2023)
+            multiplier: Fixed (1)
+        lifetime: 17.4 # 17.4 from REMDB https://remdb.nrel.gov/measures?group_id=4&component_type_id=278&actionId=1044&bcId=4261
+                       # 15 from IEA https://www.eia.gov/analysis/studies/buildings/equipcosts/pdf/appendix-a.pdf
+      - &energystar_extra_refrigerator
+        option: Misc Extra Refrigerator|EF 19.9
+        apply_logic:
+          - or:
```

Comment:
Note that none of the options with the comma in the name exist in the baseline, they do not need to be included here and I would suggest removing them. Check the baseline .tsv for the branch and include only the existing options.

Reply:
The baseline distributions can be accessed a few different ways. Easiest is probably to look at the files here resstock\project_national\housing_characteristics and see which options are included as column headers. Those are the ones existing in baseline in ResStock in our branch.

Reply:
done